### PR TITLE
PW 1584 Rearranged Links

### PR DESCRIPTION
**Pull request by zacharymorel**
## Description
Rearranged Footer links per stakeholders request.

## Testing done
Spun up FE locally and double checked links matched up in order as they do in the user story. 

## Screenshots
Screen Shot 2020-11-23 at 12 55 21 PM

## Acceptance criteria
- [x] Rearranged Footer links per stakeholders request in user story.
- [x] All links work.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/landing-pages/header-footer-data.json [hunk 1]
```diff
@@ -64,58 +64,58 @@
       "href": "https://staging.va.gov/find-forms/",
```
Comment (kelsonic) [inline src/platform/landing-pages/header-footer-data.json:64]: Weird, I wonder how come this is `staging.va.gov/find-forms` 🤔

**Review by kelsonic — APPROVED** [review]
Looks great! Nice work! 🎉

**Review by youngfreezyVA — APPROVED** [review]
LGTM

File: src/platform/landing-pages/header-footer-data.json (hunk 1 above)
Comment (zacharymorel) [reply src/platform/landing-pages/header-footer-data.json:64]: Why does that say is the subdomain `staging`?

Comment (zacharymorel) [reply src/platform/landing-pages/header-footer-data.json:64]: Regardless, I'll update it I think...

Comment (zacharymorel) [reply src/platform/landing-pages/header-footer-data.json:64]: Looks like it is that way on master....
I'll just fix it. No one ever knew.

**Review by jhonny218 — APPROVED** [review]

**Pull request merged**